userguide/directoryservice.rst

```diff
@@ -451,8 +422,7 @@ Next, only run these two commands **if** the
  service sssd start
 
 
-Finally, run these commands. Again, the :command:`echo` command should
-return a *0*:
+Finally, run these commands. The :command:`echo` will return a *0*:
```

Comment:
This changes the questionable logic of the original.  It's saying that if "echo $?" does not show a 0, something went wrong.